README.md

```diff
@@ -292,8 +296,24 @@ The CPU has many specialized registers, which we don't access directly. One of t
 ## The Math Section
 If you thought you'd get through this without doing any math, well, I'm sorry. We have to do a little bit so that we can understand what the computer is doing, because like I said, it's all just basic math underneath. Now, I promise you it won't be too hard. You may get a little confused and your brain may hurt, but just stick with me here and we'll make it through to the assembly section.
 
-### Binary
+### Hexadecimal
```

Comment:
Oooh, why'd you bring hex up first? I'm not 100% sure we need the hex section actually - I wrote it at the very beginning, but now I think it's not that useful in the guide rn, so maybe we should delete?

Reply:
I stand corrected, I was only thinking about the x86 code: https://github.com/hackclub/some-assembly-required/pull/8/files#diff-52b22214b29cd32909bb0b921201c4314652b9a42f830beaa7d018888508ffcbR43 So let's leave this in, but I'm not sure it should be first. Thoughts?

Reply:
this isn't really a good reason but i liked that after the explanation of binary it flowed into logic gates (even though there are no similarities beyond just 0 and 1 haha). i don't have a strong opinion on this though.

Reply:
Me neither - let's leave it as you have it for now and figure it out later as we fill more in :)